tools/codegen/matter_idl_plugin/matter_js_schema.jinja

```diff
@@ -0,0 +1,79 @@
+// AUTO-GENERATED with matter_idl.
+/**
+ * @license
+ * Copyright 2022 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AccessLevel, Attribute, Cluster, Command, TlvNoArguments, WritableAttribute } from "./Cluster.js";
+import { MatterCoreSpecificationV1_0 } from "../spec/Specifications.js";
+import { TlvField, TlvObject } from "../tlv/TlvObject.js";
+import { TlvEnum, TlvUInt16, TlvUInt64 } from "../tlv/TlvNumber.js";
+import { TlvString } from "../tlv/TlvString.js";
+import { TypeFromSchema } from "../tlv/TlvSchema.js";
+import { TlvBoolean } from "../tlv/TlvBoolean.js";
+
+{%- if cluster.enums %}
+
+  // Enums
+{%-   for entry in cluster.enums %}
+export const enum {{entry.name}} {
+    {%-  for field in entry.entries %}
+    {{field.name}} = {{field.code}},{%   endfor %}
+}
+{%   endfor %}{% endif -%}
+{%- if cluster.bitmaps %}
+
+  // Bitmaps
+{%-   for entry in cluster.bitmaps %}
+export const enum {{entry.name}} {
+    {%-  for field in entry.entries %}
+    {{field.name}} = {{field.code}},{%   endfor %}
+}
+{%   endfor %}{% endif -%}
+{%- if cluster.structs %}
+
+  // Structs
+{%-   for entry in cluster.structs %}{% if not entry.tag %}
+const Tlv{{entry.name}} = TlvObject({
+    {%-  for field in entry.fields %}
+    {{field.name}}:  TlvField({{field.code}}, {{field | toTlvSchemaFullType}}),{%   endfor %}
```

Comment:
could we also get min/max and the others in here? Or if not programmatically maybe as comment?